.github/workflows/load-tests.yml

```diff
@@ -9,84 +9,103 @@ on:
       - main
 
 jobs:
-  determine-service-matrix:
+  service-matrix:
```

Comment:
This is the same set of changes as `build.yml` limited to the matrix changes and no additional jobs